### PR TITLE
Update .github/labeler.yml config file

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Update .github/labeler.yml config file for new stack and file structure

## Related Issue
https://www.notion.so/efdn/Update-github-labeler-yml-config-file-28f08f039b0440639de587f10e860aea?pvs=4

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,34 +1,42 @@
 "review needed :eyes:":
   - src/**/*
+  - public/*
```
Comment (wackerow) [inline .github/labeler.yml:3]: Added `public` directory since all content has been moved into here from `src`

File: .github/labeler.yml [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
 
 "documentation :book:":
   - README.md
 
 "tooling :wrench:":
   - .github/**/*
   - src/scripts/*
+  - src/lib/*
+  - src/hooks/*
```
Comment (wackerow) [inline .github/labeler.yml:12]: Helper functions contained in these folders, so added them to tooling category. Can remove if we'd prefer to update separately:
```suggestion
```

File: .github/labeler.yml [hunk 3]
```diff
@@ -1,34 +1,42 @@
...
+  - i18n.config.json
+  - next.config.js
+  - next-i18next.config,js
+  - next-sitemap.config.js
+  - tsconfig.json
+  - .nvmrc
+  - .eslintignore
+  - .eslintrc.json
+  - .prettierignore
+  - .prettierrc
```
Comment (wackerow) [inline .github/labeler.yml:29]: Removed all Gatsby configs and added all new root level configuration files.

Would consider moving `netlify.toml` config here as well

Comment (pettinarip) [reply .github/labeler.yml:29]: > Would consider moving `netlify.toml` config here as well

I'd say yes.

**Review by pettinarip — APPROVED** [review]
Nice job @wackerow

**Pull request merged**